core/src/main/java/bio/terra/pearl/core/service/participant/EnrolleeService.java

```diff
@@ -114,6 +114,10 @@ public Enrollee loadForAdminView(Enrollee enrollee) {
         return dao.loadForAdminView(enrollee);
     }
 
+    public Enrollee loadEnrolleeDetails(Enrollee enrollee) {
```

Comment:
Will do.
Do you imagine that the dashboard will be the only user of this method?

Reply:
IMO we should generally avoid it.  It's not horrific where it's just doing loading or some other operation that's unlikely to need audit/other logic inserted, but for consistency it's best to always go through a service.  If there are cases where we didn't, it's probably just due to accidentally missing, or predating the refactor.

Reply:
yeah, I'd imagine the dashboard will be the only user.  It might be loaded in the admin tool if we ever add a "preview dashboard" feature, but that's a ways off

Reply:
Thx for that info. I will change the name rather than move to an ext service, since after reading your comment I think I misunderstood.